pages/docs/modal-system.en-US.mdx

```diff
@@ -0,0 +1,71 @@
+import { Callout } from "nextra-theme-docs";
+
+# The modal system
+
+An easier way to trigger a modal without having to have a conditional state to render it in the DOM.
+
+This is handled by the function [`showModal`](https://o3-docs.openmrs.org/docs/framework-api-reference#showmodal) defined in `esm-core` that takes care of the rendering of the modal.
+
+<Callout emoji="ℹ️" type="info">
+  For more information about how `showModal` works, check out its [implementation](https://github.com/openmrs/openmrs-esm-core/blob/01c32c2090a47a881d5f9fa39650ec4a9037aa36/packages/framework/esm-styleguide/src/modals/index.tsx).
+  The API reference can be found [here](/docs/framework-api-reference#showmodal).
+</Callout>
+
+## Usage
+
+### Implementation of the modal
+Make a seperate component for the modal, as you normally would, but instead of the top most wrapper component being the `ComposedModal`, it should be a `React.Fragment`.
```

Comment:
I'm not really sure I understand this line.

```suggestion
Make a seperate component for the modal, as you normally would, but instead of the top most wrapper component being the `ComposedModal`, it should be a `React.Fragment`.
```

Reply:
Sorry, I was trying to say that a `React.Fragment` should be at the top most of the component and not `ComposedModal`